docs/install/github-action.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Using the Github Action
+nav_order: 2
```

Comment:
`nav_order` needs to be updated to be updated on the other files in the `/install` folder. I think the following order makes sense:

0 - Docker
1 - Windows
2 - Using the Github Action
3 - Source
4 - Migrating from V1 to V2